chapters/6-analysis.tex

```diff
@@ -496,7 +497,9 @@ \section{Creating analysis outputs}
 Though it is best to be transparent about different
 specifications tried and tests performed,
 only a few will ultimately be considered ``main results''.
-These will be exported from the statistical software.
+These will be \textbf{exported}\sidenote{
+  \textbf{Export:} The creation of publication-ready representations of results.}\index{exporting results} 
```

Comment:
```suggestion
  \textbf{Exporting result:} The creation of publication-ready representations of results.}\index{exporting results} 
```
The word **export** has so many many meanings that it is weird define that general terms with just one of its meanings